terraform/gke/README.md

```diff
@@ -608,6 +608,18 @@ A [NetworkPolicy rule](../../kubernetes/decoupled/autoscaler-pkg/networkpolicy.y
 is also configured to allow traffic from the `poller` and `scaler` workloads
 (labelled with `otel-submitter:true`) to the `otel-collector` service.
 
+If the environmental variable `OTLP_COLLECTOR_URL` is not specified, the metrics
```

Comment:
@henrybell Added this since your last review.

Reply:
Final nit: "environmental" -> "environment"!

Reply:
Done